cognite/client/data_classes/capabilities.py

```diff
@@ -137,8 +148,9 @@ def load(cls, resource: dict | str) -> Self:
             return cast(
                 Self,
                 capability_cls(
-                    actions=[capability_cls.Action(action) for action in resource[name]["actions"]],
+                    actions=[capability_cls.Action.load(act, allow_unknown) for act in resource[name]["actions"]],
```

Comment:
What about just monkey patching? Would you lose information with this solution? The action is lost.

Reply:
Not sure I understood your suggestion. For the record, I just copied this one:

![IMG_1567](https://github.com/cognitedata/cognite-sdk-python/assets/8521241/20ee5914-4fe9-416f-9f4a-e673ce057a59)

Reply:
But yeah, I get your point about losing the action. Hmmm

Reply:
Switched to dynamic enums for when _unknowns_ are encountered to ensure we don't lose information. Too crazy?